tpcx_bb/cluster_configuration/cluster-startup.sh

```diff
@@ -0,0 +1,63 @@
+#IB, NVLINK, or TCP
+CLUSTER_MODE=$1
+
+# TODO: Unify interface/IP setting/getting for cluster startup
+# and scheduler file
+INTERFACE="enp134s0f1"
+
+MAX_SYSTEM_MEMORY=$(free -m | awk '/^Mem:/{print $2}')M
+DEVICE_MEMORY_LIMIT="25GB"
+POOL_SIZE="30GB"
+
+# Fill in your environment name and conda path on each node
+CONDA_ENV_NAME="rapids-tpcxbb-bsql-20200701-1501"
+CONDA_ENV_PATH="/raid/nicholasb/miniconda3/etc/profile.d/conda.sh"
+
+# TODO: Remove hard-coding of scheduler
+SCHEDULER="exp01"
+LOGDIR="/tmp/tpcx-bb-dask-logs/"
+WORKER_DIR="/tmp/tpcx-bb-dask-workers/"
+
+# Purge Dask worker and log directories
+rm -rf $LOGDIR/*
+mkdir -p $LOGDIR
+rm -rf $WORKER_DIR/*
+mkdir -p $WORKER_DIR
+
+# Purge Dask config directories
+rm -rf ~/.config/dask
+
+# Activate conda environment
+source $CONDA_ENV_PATH
+conda activate $CONDA_ENV_NAME
+
+# Dask/distributed configuration
+export DASK_DISTRIBUTED__COMM__TIMEOUTS__CONNECT="100s"
+export DASK_DISTRIBUTED__COMM__TIMEOUTS__TCP="600s"
+export DASK_DISTRIBUTED__COMM__RETRY__DELAY__MIN="1s"
+export DASK_DISTRIBUTED__COMM__RETRY__DELAY__MAX="60s"
+export DASK_DISTRIBUTED__SCHEDULER__WORK_STEALING="True"
```

Comment:
I believe this is extraneous now

Reply:
I believe this is not yet purely extraneous, but maybe @VibhuJawa can comment to clarify

Reply:
For setting work stealing on, it's definitely extraneous, as work stealing is enabled by default.

For the comm settings, I originally set these when dealing with TCP flakiness in our environment before we started using IP over Infiniband. It'd be nice to leave these, given not everyone running this benchmark will have IPoIB support.

Reply:
Ah, I misunderstood. I though this comment was referring to the entire suite. And you're right, the last one is the default. Will update

Reply:
Resolved by https://github.com/rapidsai/tpcx-bb/pull/11/commits/a3e513fdff46623185199417d418c1b8a3fdb2ed